### ISSUE TITLE
ValueError: ndarray is not C-contiguous

### ISSUE THREAD

**Issue by angelgaspar** [SimonBlanke/Hyperactive#4]
Hello there! 

In your keras_example, I have this error: 

ValueError                                Traceback (most recent call last)
<ipython-input-10-c0fa8e29da12> in <module>()
     48 
     49 opt = Hyperactive(X_train, y_train)
---> 50 opt.search(search_config, n_iter=5)

13 frames
/usr/local/lib/python3.6/dist-packages/hyperactive/memory/util.py in get_hash(object)
     14 
     15 def get_hash(object):
---> 16     return hashlib.sha1(object).hexdigest()
     17 
     18 

ValueError: ndarray is not C-contiguous

I'm using Google Colab.

**Comment by SimonBlanke** [comment]
Hello @angelgaspar 

thank you for reporting this bug!

This bug occurs because the "memory"-module of Hyperactive wants to save hash values from the numpy arrays. For that the arrays must be "C-contiguous"

So you can do one of two things to solve this.

You can set the memory to "short" in Hyperactive, so that the hash values will not be used.
```python
opt = Hyperactive(X_train, y_train, memory="short")
opt.search(search_config, n_iter=5)
```

Or you can fix this quickly by changing the order of the array before the optimization starts:

```python
X_train = np.asarray(X_train, order="C")
y_train = np.asarray(y_train, order="C")

opt = Hyperactive(X_train, y_train)
opt.search(search_config, n_iter=5)
```

Let me know if this works for you and thank you for contributing to this project. It always helps to get feedback from other developers.

**Comment by angelgaspar** [comment]
Hello @SimonBlanke 

if I install your package of PyPi:

`pip install hyperactive`

Only one solution worked fine.

I set memory to "short" in Hyperactive.

`opt = Hyperactive(X_train, y_train, memory="short")`
  `opt.search(search_config, n_iter=5)`

it worked fine for me. 


In the other case of changing the order of the array before the optimization starts.

`X_train = np.asarray(X_train, order="C")`
`y_train = np.asarray(y_train, order="C")`
`opt = Hyperactive(X_train, y_train)`
`opt.search(search_config, n_iter=5)`

I had this error:
`FileNotFoundError: [Errno 2] No such file or directory: '/Users/angelgaspar/PycharmProjects/HkerasEx/venv/lib/python3.7/site-packages/hyperactive/memory/meta_data/model_connections.json'`

However if I download your package of github and run keras_example, both solutions work fine!

**Comment by SimonBlanke** [comment]
This is a very interesting problem @angelgaspar,

The file "model_connections.json" is part of the long term memory of Hyperactive and is empty after installation. For some reason it does not get installed even though it is part of the package in Github.

If you want to run the optimization with memory="long" you can run the code snipped below just one time. This will reset the memory and create the "model_connections.json".

```python
from hyperactive.memory import reset_memory
reset_memory()
```

I will fix this issue in v2.2.3 by automatically creating the .json on startup if it is not found.

I will close this issue now, because the original problem has been resolved. If you continue to have problems with the .json you can open another issue.

Thank you again for your input! It was very helpful.
